applications/ShapeOptimizationApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -168,6 +168,7 @@ void  AddCustomUtilitiesToPython(pybind11::module& m)
         .def("ComputeUnitSurfaceNormals", &GeometryUtilities::ComputeUnitSurfaceNormals)
         .def("ProjectNodalVariableOnUnitSurfaceNormals", &GeometryUtilities::ProjectNodalVariableOnUnitSurfaceNormals)
         .def("ExtractBoundaryNodes", &GeometryUtilities::ExtractBoundaryNodes)
+        .def("ComputeProjectedDistancesToBoundingModelPart", &GeometryUtilities::ComputeProjectedDistancesToBoundingModelPart)
```

Comment:
What about shortening it to "ComputeDistancesToBoundingModelPart"? Note that the distances itself are not projected, so it might be even misleading.

Reply:
The distance is computed by searching the nearest node and then projecting the distance vector onto the normals of the bounding part. Thats why "projected" is in the name. If you think it is cleaner without, i can remove it.